go.mod

```diff
@@ -4,6 +4,7 @@ go 1.18
 
 require (
 	github.com/client9/misspell v0.3.4
+	github.com/google/go-cmp v0.6.0
```

Comment:
Could we use [`reflect.DeepEqual()`](https://pkg.go.dev/reflect#DeepEqual) to avoid adding another dependency?

Reply:
Sure, this go-cmp was already an indirect dependency so did not think it would have been to bad add, but for this simple case I think reflect works fine.

```
$ go mod graph | grep "cmp"
google.golang.org/protobuf@v1.33.0 github.com/google/go-cmp@v0.5.5
```